addons/hr_holidays/static/src/js/time_off_calendar.js

```diff
@@ -147,6 +177,39 @@ odoo.define('hr_holidays.dashboard.view_custo', function(require) {
             });
         },
 
+        _onOpenCreate: function() {
+            var self = this;
+            this._super(...arguments);
+            if(this.previousOpen) {
+                this.previousOpen.on('closed', this, function(ev){
+                    // we reload as record can be created or modified (sent, unpublished, ...)
+                    self.reload();
+                });
+                for(let i = 0; i < this.previousOpen.buttons.length; i++) {
+                    this.previousOpen.buttons[i].classes += ' timeoff_button';
+                }
+            }
+        },
+
+        _onOpenEvent: function(event) {
+            if (!this.eventOpenPopup) {
+                this._super(...arguments);
+                return;
+            }
+
+            var options = this._getFormDialogOptions(this, event);
+
+            if (this.formViewId) {
+                options.view_id = parseInt(this.formViewId);
+            }
+            let timeoffFormViewDialog = new dialogs.FormViewDialog(this, options);
+
+            for(let i = 0; i < timeoffFormViewDialog.buttons.length; i++) {
```

Comment:
Instead you could pass `dialogClass` [1] in the `options` when creating the FormViewDialog, then your CSS can be:

```scss
.whatever_you_choose_in_dialogClass {
    .o_btn_remove.timeoff_button {
        margin-left: 0px;
        padding-left: 4px;
        padding-right: 4px;
    }
    .btn.o_form_button_cancel.timeoff_button {
        padding-left: 4px;
        padding-right: 4px;
    }
}
```

Maybe something similar is somehow possible to do in `_onOpenCreate()` :-)

However, it's strange that you need to manually add padding on the buttons :thinking: 

[1] : https://github.com/odoo/odoo/blob/master/addons/web/static/src/legacy/js/views/view_dialogs.js#L32

Reply:
PR updated to remove the custom CSS for the buttons. Thanks :-)